ckanext/xroad_integration/logic/action.py

```diff
@@ -433,7 +433,7 @@ def fetch_xroad_errors(context, data_dict):
     for harvest_source in harvest_sources:
         source_title = harvest_source.get('title', '')
 
-        if data_dict.get('since'):
+        if "since" in data_dict and data_dict.get('since') is not None:
```

Comment:
`if None` equals false, so `data_dict.get('since')` and `data_dict.get('since') is not None` checks the same thing

Reply:
Fixed that one as well